invenio_oaiserver/errors.py

```diff
@@ -29,3 +29,7 @@
 
 class OAIBadMetadataFormatError(Exception):
     """Metadata format required doesn't exist."""
+
+
+class OAISetNotSupported(Exception):
```

Comment:
Can we make the name a bit more clear with `OAISet(Update|Modification)Error`?

(exception should end with `Error`)